modules/virt-creating-template-redhat-web.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+[id="virt-creating-template-redhat-web_{context}"]
+= Creating a virtual machine template from a Red Hat template in the web console
+
+Select a virtual machine template provided by Red Hat and customize the template by adding a boot source and saving the template. The customized virtual machine template is available for you to create multiple virtual machines.
+
+.Procedure
+
+. In the {VirtProductName} console, click *Workloads* -> *Virtualization* from the side menu.
+. Click the *Templates* tab.
+. Identify the Red Hat template that you want to use for creating a virtual machine template. Click *Add source* for that template.
```

Comment:
If the template already have source the "Add source" button will be replaced with the source provider name or default "available" if no provider name was provided. Can we say something like "If source is not provided ... "

Reply:
Expanded this task to instruct the user what should be done if 1) the red hat template was configured with a boot source and shows Available or 2) if the red hat template was not configured with a boot source and shows Add source.